src/main/drivers/exti.c

```diff
@@ -184,6 +184,38 @@ void EXTIEnable(IO_t io, bool enable)
 #endif
 }
 
+void EXTISetTrigger(IO_t io, EXTITrigger_TypeDef trigger)
+{
+uint32_t extiMask;
+
+#if defined(STM32F1) || defined(STM32F4) || defined(STM32F7)
```

Comment:
Yikes!!! I've totally forgotten that I added this part of the code; it was so long ago 🤣